nservicebus/azure-service-bus/configuration/azureservicebusqueueconfig.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Azure Service Bus Transport Configuration
+title: Azure Service Bus Queue Config Section
```

Comment:
For menu shortening to work, child node title has to begin with parent`s node title.
So `Azure Service Bus Transport Queue Configuration Section`